tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -62,3 +66,134 @@ def get_framework_results(self, inputs_dict, model_path):
         else:
             # get results from tflite
             return get_tflite_results(self.use_legacy_frontend, inputs_dict, model_path)
+
+    def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp_dir,
+              use_legacy_frontend=False, infer_timeout=60, **kwargs):
+        """
+        :param enabled_transforms/disabled_transforms: string with idxs of transforms that should be enabled/disabled.
```

Comment:
This docstring doesn't correspond the method signature

Reply:
I removed it at all.